google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/JsonStreamWriterTest.java

```diff
@@ -603,4 +606,37 @@ public void testFlowControlSettingNoLimitBehavior() throws Exception {
       appendFuture.get();
     }
   }
+
+  @Test
+  public void testMultipleAppendSerializtionErrors()
+      throws DescriptorValidationException, IOException, InterruptedException {
+    FooType expectedProto = FooType.newBuilder().setFoo("allen").build();
+    JSONObject foo = new JSONObject();
```

Comment:
tiny optional nit: can you add comments on why each of these is valid/invalid

Reply:
done